src/e2e/java/io/proximax/sdk/E2EContractTest.java

```diff
@@ -76,17 +82,17 @@ void closeDown() {
 
    @Test
    void prepare() {
-      signup(contract.getAddress());
-   }
-
-   @Test
-   void test01ChangeContractAccountToMultisig() {
-
+      signup(accContract.getAddress());
+      logger.info("Going to create contract for {}", accContract);
+      logger.info("customers {}", customer);
+      logger.info("executors {}, {}", executor1, executor2);
+      logger.info("validators {}, {}", validator1, validator2);
    }
 
    @Test
    void test02CreateContract() {
-      // prepre transaction
+      logger.info("Creating constract");
```

Comment:
~~Creating constract~~ Creating contract

Reply:
thanks, will fix that typo with next commit I make :)

Reply:
noted this down in issue #44 where I will be making general improvements to tests